enh: Support search by case number

poyea opened the pull request:
Fixes #191 for now.
This patch enables search by case number in cases page. The number of
entries for number is limited to 3.

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] i18n translation has been added / updated (for bug fixes / features)


* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ..., provide screenshots if necessary)


* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)


* **Other information**:

poyea committed:
enh: Support search by case number
Fixes #191 for now.
This patch enables search by case number in cases page. The number of
entries for number is limited to 3.

nandiheath commented on src/utils/search.js:
```diff
@@ -4,6 +4,24 @@ import _uniq from "lodash.uniq"
+
+export const createCasesOptions = (edges, field) => {
+  return edges
+    .map(({ node }) => node.case_no)
```
should be `.map(({ node }) => node[field] || '' )` ?

nandiheath reviewed (changes requested):
Awesome! Sorry for my messy code but you can still add function on it!

poyea committed:
Maintain order when searching by number

poyea commented:
> Awesome! Sorry for my messy code but you can still add function on it!

I just deleted that for simplicity. haha

> should be `.map(({ node }) => node[field] || '' )` ?

With that context it's the same, but I changed to this in the newer `createDedupOptions()` . It also facilitates the search.
